docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -208,6 +208,8 @@ index can be <<indices-delete-index,removed>>. You may then consider setting
 <<indices-add-alias,Create Alias>> against the destination index for the source 
 index's name to point to it for continuity. 
 
+See link:https://www.youtube.com/watch?v=sHyNYnwbYro[this video] for example
```

Comment:
```suggestion
See this https://www.youtube.com/watch?v=sHyNYnwbYro[fixing shard sizes video] for an example troubleshooting walkthrough.
```

@stefnestor two minor remarks:

- I'd like to remove `link:` because it's superfluous
- I'd prefer for the link text to be a better indicator of the video content. (This comment applies to your other PR too 🙏 )

The descriptive link text is better for SEO and A11Y (screen readers).
This will make it easier to update/remove these in the future if this content becomes stale or unavailable.